apps/registry/pages/api/formatters/template.js

```diff
@@ -28,7 +30,7 @@ export const THEMES = {
   spartan: require('jsonresume-theme-spartan'),
   stackoverflowed: require('jsonresume-theme-stackoverflowed'),
   stackoverflow: require('jsonresume-theme-stackoverflow'),
-  standard: require('jsonresume-theme-standard'),
```

Comment:
Idk what changed but now it wants to properly import the package. I see no harm in doing that.